Mathlib/CategoryTheory/Category/Bipointed.lean

```diff
@@ -27,9 +27,11 @@ universe u
 
 variable {α β : Type _}
 
+set_option linter.uppercaseLean3 false
+
 /-- The category of bipointed types. -/
 structure Bipointed : Type (u + 1) where
-  pt : Type u
+  X : Type u
```

Comment:
Is there a reason this was renamed from `pt` to `X`? If so, can you add an `#align` that records the mapping into the Lean3 name?

Reply:
Hmm, this looks like a mathport bug? It contains:

https://github.com/leanprover-community/mathlib3port/blob/master/Mathbin/CategoryTheory/Category/Bipointed.lean#L30-L34

but in Lean3 the field is called `X`. @digama0, do you know what's going on here?